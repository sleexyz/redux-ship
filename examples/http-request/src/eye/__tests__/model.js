// @flow
import * as EyeModel from '../model';

const commits = [{
  type: 'LoadStart',
}, {
  type: 'LoadSuccess',
  color: 'red',
}]

test('model', () => {
  commits.forEach(commit => {
    expect(EyeModel.reduce(EyeModel.initialState, commit)).toMatchSnapshot();
  });
});
