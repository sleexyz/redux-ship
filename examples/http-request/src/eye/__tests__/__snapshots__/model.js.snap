exports[`test model 1`] = `
Object {
  "color": null,
  "isLoading": true,
}
`;

exports[`test model 2`] = `
Object {
  "color": "red",
  "isLoading": false,
}
`;
